src/contrib/cluster/Akka.Cluster.Sharding/Internal/RememberEntityStarter.cs

```diff
@@ -93,7 +93,7 @@ public RememberEntityStarter(
             Timers.StartPeriodicTimer("retry", ResendUnAcked.Instance, settings.TuningParameters.RetryInterval);
         }
 
-        public ITimerScheduler Timers { get; set; }
+        public ITimerScheduler Timers { get; set; } = null!;
```

Comment:
Fixed a nullability warning here. This value always gets set by the Akka.NET infrastructure.